src/assets/style/variables/typography.scss

```diff
@@ -29,10 +29,16 @@ h5 {
 
 // Text sizes
 p,
-span {
+span,
```

Comment:
I think short multiply marks can be in one line instead of multiply lines

Reply:
#Prettier